paddycarey/mopidy-webhooks
# Overhaul for modern python+mopidy

- Support python3
- Use standard mopidy code quality tools
- Replace travis with github actions
- Rewrite tests to use pytest
- Add initial type checking
- Modify configuration handling slightly
- Other minor tweaks
- Update readme
  - Reformat to make it easier to read outside of github
  - Update links
  - Remove changelog from readme. It's of little value in its current
    form, and should be continued as a standalone file if still deemed
    necessary in future.
